src/pywrapper.cpp

```diff
@@ -290,6 +291,17 @@ bool PyWrapper::convert(void* in_, MultiTypeValue& out)
                 out.vf.push_back(val);
                 out.type = MultiTypeValue::Type::VECTOR_FLOAT;
             }
+	    if(PyBytes_Check(el) && (out.type == MultiTypeValue::Type::NONE || out.type == MultiTypeValue::Type::VECTOR_STRING ||
+				     out.type == MultiTypeValue::Type::VECTOR_FLOAT /* how not to end here ? */
```

Comment:
Is VECTOR_FLOAT appropriate here?

Reply:
Nope: result of the default setting of above